frontend/app-development/features/appPublish/components/utils.test.ts

```diff
@@ -0,0 +1,25 @@
+import { versionNameValid } from './utils';
+import type { IRelease } from '../../../sharedResources/appRelease/types';
+import { BuildResult, BuildStatus } from '../../../sharedResources/appRelease/types';
+
+const releases: IRelease[] = [
+  {
+    tagName: 'existing-version-tag',
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
Hvorfor `@ts-ignore` her?

Reply:
For å slippe og sette opp hele datamodellen for test. Her er det eksponert masse data som ikke er i bruk.